deid/utils/actions.py

```diff
@@ -66,10 +67,28 @@ def parse_value(dicom, value, item=None, field=None, funcs=None):
             # item is the lookup, value from the recipe, and field
             # The field is an entire dicom element object
             return item[value_option](dicom=dicom, value=value, field=field, item=item)
-
-        bot.warning("%s is not a valid value type, skipping." % (value_type))
-        return None
-    return value
+        else:
+            bot.warning("%s is not a valid value type, skipping." % (value_type))
```

Comment:
If we want to start using format strings across the code base, maybe it makes sense to only add string parsing using them.
```suggestion
            bot.warning(f"{value_type} is not a valid value type, skipping.")
```